portfolio/src/main/webapp/index.html

```diff
@@ -7,13 +7,70 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+     <div id="content">
+       <h1>Jonathan Peña</h1> 
+       <p>California State Polytechnic University, Pomona</p>
+       <p>B.A. Computer Science</p>
+      
+       <a href="https://www.linkedin.com/in/jonathanpe%C3%B1a">Linkedin</a>
+       <a href="https://github.com/JonathanPena17">GitHub</a>
+     
+     </div>
+      
+    <div class ="aboutme">
+     
+      <h2>About Me</h2>
+      
+      <img src="/images/Headshot.jpeg" width = "300" height = "300" class="rounded" />
+        
+      <p>. Hello! My name is Jonathan Pena and I am a rising third-year student attending Cal Poly Pomona. 
+      I am a first-generation student and a son of immigrant parents. I am passionate about science and space. 
+      Hopefully, one day, I can be a part of the Artemis Generation and help progress human exploration. </p>
+    
+      <p>  Outside of school, I have little to no free time. The majority of the time I spend working for my family's 
+      business which is sports field maintenance and renovation. Yet the times when I have a time slot, I spend that 
+      time resting or watching educational videos/podcasts related to cars or science.</p>
+      
+      <p>Click here to get a random fact about me!</p>
+      <button onclick="addRandomGreeting()">Fact</button>
       <div id="greeting-container"></div>
+    
+      <h2> Favorite Artists</h2>
+
+      <a href="https://open.spotify.com/artist/5K4W6rqBFWDnAN6FQUkS6x?si=9ZL9xGjeThi4CaYv55MMigQ">
+        <img alt="Qries" src="/images/Kanye.jpeg" width="200" height="200" class="rounded-corners" style="float:left">
```

Comment:
alt tag of image displays when the picture has trouble loading, you can make it more descriptive like the artist names.